tensorflow/stream_executor/rocm/rocm_blas.cc

```diff
@@ -2324,8 +2327,15 @@ bool ROCMBlas::DoBlasTrsmBatched(Stream *stream, blas::Side side,
                                  float alpha, const DeviceMemory<float *> &as,
                                  int lda, DeviceMemory<float *> *bs, int ldb,
                                  int batch_count) {
-  // ROCM TODO: properly implement the interface
-  return false;
+  blas_log("DoBlasTrsmBatched");
+  LOG(INFO) << "DoBlasTrsmBatched";
+  VLOG(1) << absl::StreamFormat(
+      "doing rocBLAS trsm Batched<float>");
+  return DoBlasInternal(wrap::rocblas_strsm_batched, stream,
+                        true /* = pointer_mode_host */, ROCMBlasSide(side),
```

Comment:
`true`?  ... thought `true` meant the pointer location is on the host

Reply:
hmm..yeah. It's set to `true` everywhere else in the file, so I kept it the same. I'll look into that more to be sure that's right.